specification/peering/resource-manager/Microsoft.Peering/stable/2021-01-01/peering.json

```diff
@@ -3398,11 +3405,19 @@
       "type": "object",
       "properties": {
         "peeringServiceLocation": {
-          "description": "The PeeringServiceLocation of the Customer.",
+          "description": "The location (state/province) of the customer.",
           "type": "string"
         },
         "peeringServiceProvider": {
-          "description": "The MAPS Provider Name.",
+          "description": "The name of the service provider.",
+          "type": "string"
+        },
+        "providerPrimaryPeeringLocation": {
+          "description": "The primary peering (Microsoft/service provider) location to be used for customer traffic.",
+          "type": "string"
+        },
+        "providerBackupPeeringLocation": {
```

Comment:
Reviewed the latest 3 commits to look at the delta changes from previous API version. Changes LGTM. signing off from ARM side.